Tests/Migration/DataPipelineMigrationAssistantTests.swift

```diff
@@ -12,16 +11,24 @@ class DataPipelineMigrationAssistantTests: UnitTest {
     // that have gone undiscovered in the code when `CustomerIO` passes a request to `DataPipelineImplementation`.
     private let profileStoreMock = ProfileStoreMock()
     private let backgroundQueueMock = QueueMock()
-
-    private var migrationAssistant: DataPipelineMigrationAssistant { diGraph.dataPipelineMigrationAssistant }
+    private let migrationHandler = DataPipelineMigrationActionMock()
     private var queueStorage: QueueStorage { diGraph.queueStorage }
-
+    public var migrationAssistant: DataPipelineMigrationAssistant!
```

Comment:
can we avoid making it nullable? we have been trying to remove it from test cases as they tend to introduce flakiness in tests.